server/server.js

```diff
@@ -22,16 +33,27 @@ export default function server(app: express, options: any):express {
     app.use("/kue", queueUiRouter({ hostSecret, queueAgent: queue }));
   }
 
-  app.use((req, res, next) => {
+  const applicationDirectory = path.dirname(
+    path.join(require.main.filename, "..")
+  );
+
+  // any subdirectories can serve custom static assets
+  app.use(express.static(`${applicationDirectory}/connectors`));
```

Comment:
This is to serve the static assets for each of the connectors we want to break out from the base